src/utils.js

```diff
@@ -204,6 +211,7 @@ const getItemBaseValue = itemId => itemsMap[itemId].value
  * @returns {number}
  */
 export const getItemCurrentValue = ({ id }, valueAdjustments) =>
+  // should valueAdjustments have a "global" adjustment or something for any permanent global increases?
```

Comment:
Since it seems like we're getting close to the end with this PR, please be sure to remove this comment (unless it's something you feel is unresolved). 🙂

Reply:
another good catch! we resolved this with the discussion about these being only for market adjustments 👍